<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RedPitayaControlDialog</class>
 <widget class="QDialog" name="RedPitayaControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>151</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RedPitayaControlDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutMain">
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutControl">
       <item>
        <widget class="QGroupBox" name="groupBoxHostName">
         <property name="title">
          <string>Host name</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QComboBox" name="comboBoxHostName">
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>rp-f05825.local</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>192.168.1.101</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>192.168.1.200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>192.168.1.202</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxLed">
         <property name="title">
          <string>LED</string>
         </property>
         <layout class="QGridLayout" name="gridLayoutLed"/>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>RedPitayaControlDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
